microgen/shape/tpms.py

```diff
@@ -106,26 +106,12 @@ def __init__(
             raise ValueError("density must be between 0 and 1")
         self.density = density
 
-    def _max_density(
-        self,
-        part_type: Literal["sheet", "lower skeletal", "upper skeletal"],
-        resolution: Optional[int] = None,
-    ) -> float:
-        if part_type == "sheet":
-            return 1.0
-        tpms = Tpms(
-            surface_function=self.surface_function,
-            offset=0.0,
-            resolution=resolution if resolution is not None else self.resolution,
-        )
-        return tpms.generateVtk(type_part=part_type).volume / tpms.grid.volume
-
     @classmethod
     def offset_from_density(
         cls,
         surface_function: Callable[[np.ndarray, np.ndarray, np.ndarray], np.ndarray],
         part_type: Literal["sheet", "lower skeletal", "upper skeletal"],
-        density: float | Literal["max"] = "max",
+        density: float = 0.3,
```

Comment:
Do we really need a default value for this method ? If yes, maybe put 0.5 to be consistent with documentation